practices/code-review.md

```diff
@@ -11,21 +11,23 @@ While we can support "traditional" Code Reviews, we routinely review via **pull
 We leave it to the requester to merge their own changes when they are ready, because sometimes the requester might not want their work merged immediately.
 
 
-## Why
-
-1. PR reviews are cross-team;
-    * allows everyone to learn from their colleagues.
-    * allows greater visibility of other teams work.
-    * FED works encompasses many areas of expertise, and no-one is expert in more than 2 or 3 areas.
-    * facilitates people moving to other teams as they are more familiar with the codebase.
-    * helps prevent knowledge silos.
-    * helps identify areas where code re-use may be of benefit ("should that be a component?").
-    * if the code requires so much domain-specifc knowledge it cannot be understood by other teams, maybe that code needs refactoring?
-    * generally aids onboarding and training.
-1. Ensure that any changes to front-end code conforms to the same standards and guidelines.
-1. Fosters discussion and collaboration.
-1. Fosters caution and code quality over temporary hacks.
+## Why do them?
+
+**PR reviews are cross-team helping to foster discussion and feedback**
```

Comment:
`cross-team, thereby helping` ?

Reply:
Not sure what you mean here -- please clarify.